packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -223,7 +224,40 @@ class TextStyleTween extends Tween<TextStyle> {
 ///
 /// Which properties are animated is left up to the subclass. Subclasses' States
 /// must extend [ImplicitlyAnimatedWidgetState] and provide a way to visit the
-/// relevant fields to animate.
+/// relevant fields to animate. The "see also" section below lists a number of
+/// concrete subclasses that ship with the framework (their names usually start
+/// with `Animated`).
+///
+/// ## Relationship to [AnimatedWidget]s
+///
+/// In contrast to [AnimatedWidget]s (and its subclasses),
+/// [ImplicitlyAnimatedWidget]s instantiate and manage their own
+/// [AnimationController]s, which may be inefficient for multiple parallel
+/// animations. For those cases, consider instantiating an explicit
+/// [AnimationController] that is passed to (subclasses of) [AnimatedWidget]s
+/// instead of using (subclasses of) [ImplicitlyAnimatedWidget]s.
+///
+/// See also:
```

Comment:
Maybe as we develop the animation documentation further this could link to a catalog of implicitly animated widgets with colorful assets etc. instead of a long list here. I think the list if perfect for visibility right now. Just food for thought, or something to return to later. :)

Reply:
Yeah, that would be cool!